src/io/inputs/model-converter/include/antares/io/inputs/model-converter/modelConverter.h

```diff
@@ -47,7 +47,49 @@ namespace Antares::IO::Inputs::ModelConverter
 class UnknownTypeException: public std::runtime_error
```

Comment:
Do we have unit test that catch each exceptions ?

Reply:
Yes